shared/featureconfig/config.go

```diff
@@ -48,7 +48,6 @@ type Flags struct {
 	EnableNoise                                bool // EnableNoise enables the beacon node to use NOISE instead of SECIO when performing a handshake with another peer.
 	DontPruneStateStartUp                      bool // DontPruneStateStartUp disables pruning state upon beacon node start up.
 	NewStateMgmt                               bool // NewStateMgmt enables the new state mgmt service.
-	DisableInitSyncQueue                       bool // DisableInitSyncQueue disables the new initial sync implementation.
```

Comment:
Should you delete the flag entirely?

Reply:
I think we should. We still have `deprecatedDisableInitSyncQueueFlag`, so that users are notified if they use it. But this very string var isn't used anywhere.